TechTalk.SpecFlow/Assist/TableExtensionMethods.cs

```diff
@@ -49,10 +49,12 @@ public static IEnumerable<T> CreateSet<T>(this Table table)
 
         public static IEnumerable<T> CreateSet<T>(this Table table, InstanceCreationOptions creationOptions)
         {
-            var list = new List<T>();
+            int count = table.Rows.Count();

```

Comment:
```suggestion
            int count = table.Rows.Count;

```
Rows has a Count property that doesn't rely on Enumerable.Count()